### PR TITLE
Sentry integration

### PR DESCRIPTION
**Pull request by xepozz**
This is an example of integration with [Sentry](https://sentry.io/).

**Pay attention**, you should add Sentry DSN into [config file](https://github.com/yiisoft/yii-sentry/blob/1d8e0642c81a7df1e978effa6cf9b4c8c7ba1bf5/config/params.php#L9) config/packages/yiisoft/yii-sentry/params.php

### CODE REVIEW

File: config/routes.php [hunk 1]
```diff
@@ -12,9 +12,11 @@
 use Yiisoft\Router\Route;
 use Yiisoft\Swagger\Middleware\SwaggerJson;
 use Yiisoft\Swagger\Middleware\SwaggerUi;
+use Yiisoft\Yii\Sentry\SentryMiddleware;
 
 return [
     Route::get('/')
+        ->middleware(SentryMiddleware::class)
```
Comment (samdark) [inline config/routes.php:19]: Shouldn't it be in the app middlware instead? https://github.com/yiisoft/app-api/blob/master/config/web/application.php#L21

Comment (xepozz) [reply config/routes.php:19]: Of course. I just didn't find that place :)

**Review by samdark — APPROVED** [review]

**Comment by BoShurik** [comment]
Will sentry handle cli context in that case?

**Comment by samdark** [comment]
@BoShurik nope.

**Comment by samdark** [comment]
See https://github.com/yiisoft/yii-sentry/issues/3

**Comment by xepozz** [comment]
It depends on https://github.com/yiisoft/app-api/pull/86

**Comment by vjik** [comment]
Not need for application template

**Pull request closed without merging**